modules/terrain/src/lib/parse-terrain.js

```diff
@@ -89,15 +91,32 @@ function getMartiniTileMesh(terrainImage, terrainOptions) {
       boundingBox: getMeshBoundingBox(attributes)
     },
     mode: 4, // TRIANGLES
-    indices: {value: triangles, size: 1},
+    indices: { value: Uint32Array.from(triangles), size: 1 },
     attributes
   };
 }
 
+function getMartiniTileMesh(meshMaxError, width, terrain) {
```

Comment:
Can we add JSDoc headers to each function, just a one liner explaining what the function does. Even better if we add @param with types.